test/challenges-platform/services/submissions-service.test.ts

```diff
@@ -32,6 +36,25 @@ describe("SubmissionService", () => {
         expect(result.val.toString()).toBe("Error: Failed to find challenge");
       });
     });
+    describe("when challenge is deleted", () => {
+      it("returns an error", async () => {
+        const factory = await challengeFactory();
+        // Delete the challenge
+        const challenge = await ChallengesService.destroy(factory.uuid);
```

Comment:
Instead of having to fire the challenges service, the challengeFactory should take `deleted` as a parameter